### ISSUE TITLE
Wire Stools do not respect Prop Protection

### ISSUE THREAD

**Issue by Nebual** [wiremod/wire#62]
All wire stools that can update their SENTs (thrusters, constant values, rangers, etc) currently only allow updating if ENT.pl == tool:GetOwner(). As a result of this:
- If you reconnect to a server, your old wire entities cannot be updated as disconnecting causes ENT.pl to become NULL (so shooting old sents will just dumbly spawn a new one on top)
- Your prop protection friends/admins cannot update your sents

If we assume all reasonable servers have some degree of prop protection, then it should be left up to PP to decide if someone CanTool an entity, not up to the stool itself. If you dislike this assumption, consider that on a server lacking prop protection, anyone can just removergun your wire SENTs anyway, and manually remake them with the updated settings (or just remove your whole contraption; all serious servers will have prop protection).

It would be a simple commit (which I have prepared) to just remove these builtin ownership checks for all stools (Except expression2, which has its own system in place that respects prop protection).

tl;dr: Ownership checks belong in prop protection, not in stools themselves.
